webpack/react_app/components/TargetingHosts/__tests__/__snapshots__/TargetingHosts.test.js.snap

```diff
@@ -74,7 +74,15 @@ exports[`TargetingHosts renders with loading 1`] = `
           </th>
         </tr>
       </thead>
-      <tbody />
+      <tbody>
+        <tr>
+          <td
+            colSpan="3"
+          >
+            No hosts found.
```

Comment:
interesting, in the fixtures there is one item

Reply:
It's correct, for loading scenario `hosts array` is empty:
```
  'renders with loading': {
    apiStatus: 'PENDING',
    items: [],
  },
```